examples/user-plasticity.yaml

```diff
@@ -0,0 +1,17 @@
+name: Plastic deformation of a perforated strip
+authors:
+  - Clemens Verhoosel
+description: |
+  This example considers the plastic deformation of a perforated strip. More
+  information regarding this prototypical nonlinear solid mechanics problem
+  can be found in, for example, "The Finite Element Method for Solid and
+  Structural Mechanics" by Zienkiewicz and Taylor.
+repository: https://github.com/CVerhoosel/nutils-plasticity.git
+branch: release/6
```

Comment:
For user contributed examples the branch should actually be a specific commit and the commit should belong to the given repository. In this case the commit should be `23bd1c0e96c90bda24a2e82394bb026fc9b87fcf`. We have the following reasons for only accepting commits (or more general: immutable code):

* We'd like to review the code before accepting an example. By requiring a commit, the owner of the example should send a PR to update the example, which gives us the opportunity to review the changes.
* We consider adding the examples to the Nutils CI tests. If an example breaks for reasons other than a change in the Nutils library, this would add noise to the test results and might lead to ignoring the results altogether.
* We host the list of examples and do not want to refer to content that we do not approve.

I have a couple of suggestions for your example, which I will post to your repository.